docs/pages/database-access/troubleshooting.mdx

```diff
@@ -39,3 +39,27 @@ details and examples:
 - [Redis Cluster](./guides/redis-cluster.mdx#step-46-set-up-mutual-tls)
 
 After the new certificate is issued, update your database to make it take effect.
+
+### Access to db denied
+
+Attempts to connect to the database fail with an error message
+similar to: **"access to db denied"**.
+
+**Solution:** Configure the database instance and Teleport role-based access control (RBAC) to allow the user 
+access.
+
+Check that the database user and database name (aka schema) exist in the database instance.
```

Comment:
```suggestion
Check that the database user and database name (aka schema) you are trying to access via `tsh` exist in the database instance.
```
To indicate where the database user and name come from (assuming this is correct)